src/containers/paper-canvas.jsx

```diff
@@ -81,7 +81,14 @@ class PaperCanvas extends React.Component {
     }
     importSvg (svg, rotationCenterX, rotationCenterY) {
         const paperCanvas = this;
-        paper.project.importSVG(svg, {
+        let processedSvg = svg.split('svg:').join('');
+        if (processedSvg.indexOf('xmlns=') === -1) {
```

Comment:
Is there any chance this could match, say, an SVG with a text element that contains "xmlns="?

![Scratch 2.0 paint editor including "xmlns=" text](https://user-images.githubusercontent.com/9948030/33854318-338d9618-deb9-11e7-886a-83725e68b613.png)

Reply:
Haha good catch. I rewrote the process to only check within the svg tags attributes. @fsih can you re-review.